test/api_test.js

```diff
@@ -130,4 +143,25 @@ suite('api', () => {
     result = await helper.github.isInstalledFor('unknownOwner', 'unknownRepo');
     assert.deepEqual(result, {installed: false});
   });
+
+  test('build badges', async function() {
+    await request.get('http://localhost:60415/v1/badge/abc123/coolRepo/master').end((err, res) => {
+      err ? console.log(err) : assert.equal(res.headers['content-length'], 928);
```

Comment:
If there's an error, it will get sent to the console -- but the test will still succeed.  It's probably better to throw the error if it occurs.

Also, this is an odd combination of callback and `await`.  @imbstack I think you know more about superagent-promise than I do? Is this right?